src/it/scala/it/pagopa/interop/catalogmanagement/ItSpecHelper.scala

```diff
@@ -78,12 +79,19 @@ trait ItSpecHelper
 
     val eServiceApi =
       new EServiceApi(
-        new EServiceApiServiceImpl(system, sharding, persistentEntity, mockUUIDSupplier, mockFileManager),
+        new EServiceApiServiceImpl(
+          system,
+          sharding,
+          persistentEntity,
+          mockUUIDSupplier,
+          mockDateTimeSupplier,
+          mockFileManager
+        ),
         eServiceApiMarshaller,
         wrappingDirective
       )
 
-    if (ApplicationConfiguration.projectionsEnabled) initProjections()
+    if (ApplicationConfiguration.projectionsEnabled) initProjections(system.executionContext)
```

Comment:
We used the `blockingEC` elsewhere, but probably, using akka's thread pool is fine too, we need to check (I'll do it) where and how this ec is propagated downstream and who uses this ec and for which reason.

Reply:
🙇‍♂️